DSCResources/MSFT_NetworkTeam/MSFT_NetworkTeam.psm1

```diff
@@ -0,0 +1,243 @@
+﻿# Fallback message strings in en-US
+DATA localizedData
+{
+    # same as culture = "en-US"
+ConvertFrom-StringData @'    
+    getTeamInfo=Getting network team information for {0}.
+    foundTeam=Found a network team with name {0}.
+    teamMembersExist=Members in the network team {0} exist as per the configuration.
+    teamNotFound=Network team with name {0} not found.
+    lbAlgoDifferent=Load Balancing Algo is different from the requested {0} algo.
+    teamingModeDifferent=Teaming mode is different from the requested {0} mode.
+    modifyTeam=Modifying the network team named {0}.
+    membersDifferent=Members within the team named {0} are different from that requested in the configuration.
+    removingMembers=Removing members {0} not specified in the configuration.
+    addingMembers=Adding members {0} that are not a part of the team configuration.
+    createTeam=Creating a network team with the name {0}.
+    removeTeam=Removing a network team with the name {0}.
+    teamExistsNoAction=Network team with name {0} exists. No action needed.
+    teamExistsWithDifferentConfig=Network team with name {0} exists but with different configuration. This will be modified.
+    teamDoesNotExistShouldCreate=Network team with name {0} does not exist. It will be created.
+    teamExistsShouldRemove=Network team with name {0} exists. It will be removed.
+    teamDoesNotExistNoAction=Network team with name {0} does not exist. No action needed.
+    waitingForTeam=Waiting for network team status to change to up.
+    createdNetTeam=Network Team was created successfully.
+    failedToCreateTeam=Network team with specific configuration failed to changed to up state within timeout period of 120 seconds.
+'@
+}
+
+if (Test-Path "${PSScriptRoot}\${PSUICulture}")
```

Comment:
Maybe here we could have a Test-Path (whatever) and an else Import en-US so we can just remove the lines above. Instead of having two different string sets (here and a seperate file).  This chunk might make a good standard to go off of too (what do you think @PlagueHO?)